src/Main.java

```diff
@@ -1,5 +1,63 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("task 1");
+        for (int i = 0; i <= 10; i++) {
+            System.out.println(i);
+        }
+        System.out.println(" ");
+        System.out.println("task 2");
+        for (int i = 10; i >= 0; i--) {
```

Comment:
```suggestion
        for (int i = 10; i >= 1; i--) {
```
от 10 до 1 по заданию требовалось